download_action.go

```diff
@@ -117,5 +129,17 @@ func (d *DownloadAction) Run(context *DebosContext) error {
 		return fmt.Errorf("Unsupported URL is provided: '%s'", url.String())
 	}
 
+	if d.Unpack == true {
+		targetdir := context.downloaddir
+		if d.Makedir == true {
+			targetdir = filename + ".d"
+			if err := os.MkdirAll(targetdir, 0755); err != nil {
+				return err
+			}
+		}
+
+		return UnpackTarArchive(filename, targetdir, d.Compression, "--no-same-owner", "--no-same-permissions")
```

Comment:
--no-same-owner and no-same-permissions are interesting. I guess it's the right thing  to do indeed :). Does make me wonder if the tar refactoring was right by exposing tar options directly. Otoh it's internal api so that's fine ;)